config.md

```diff
@@ -328,7 +328,7 @@ Hooks allow for the configuration of custom actions related to the [lifecycle](r
     Entries in the array contain the following properties:
     * **`path`** (string, REQUIRED) with similar semantics to [IEEE Std 1003.1-2001 `execv`'s *path*][ieee-1003.1-2001-xsh-exec].
       This specification extends the IEEE standard in that **`path`** MUST be absolute.
-    * **`args`** (array of strings, REQUIRED) with the same semantics as [IEEE Std 1003.1-2001 `execv`'s *argv*][ieee-1003.1-2001-xsh-exec].
+    * **`args`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2001 `execv`'s *argv*][ieee-1003.1-2001-xsh-exec].
```

Comment:
+1 for keeping consistency for `args`'s semantics between `process` and `hooks`, either current version for process or go semantics works for me.